gearmand/requirements.txt

```diff
@@ -4,5 +4,10 @@
 #
 #    pip-compile --generate-hashes --output-file requirements.txt requirements.in
 #
-gearman==2.0.2 \
+
+gearman==2.0.2 ; sys_platform != "win32" and python_version < "3.0" \
     --hash=sha256:2b1876a60e32e24bed4c6d187898274de6409bbfa942bcbe3512efdf2aed0ec9
+
+python3-gearman==0.1.0 ; sys_platform != "win32" and python_version > "3.0" \
```

Comment:
Are we bumping the req in the root agent-req file now or when release happens?

Reply:
Let's do it now pls

Reply:
@nmuesch I bumped the requirements file, can you rereview